advanced-achievements-plugin/src/test/java/com/hm/achievement/command/pagination/CommandPaginationTest.java

```diff
@@ -50,6 +51,7 @@ public void setUp() throws Exception {
 	}
 
 	@Test
+	@Ignore
```

Comment:
The tests fail though thats why i disabled them
`java.io.FileNotFoundException: C:\Git%20repos\AdvancedAchievements\advanced-achievements-plugin\target\classes\lang.yml (The system cannot find the path specified)`
which is weird because the file is there, any suggestions on why they fail ?

Reply:
That's indeed odd. Are you running the tests from an IDE or the command line?

Reply:
IDE when trying to build with maven, also tried to just run the test from the IDE.

Reply:
Hmmm, there seems to be a problem with how paths are resolved on Windows when there are spaces in the directory. I'll look into it, but for now I suggest simply checking out the project in a folder that has no spaces. :wink:

Reply:
This was hopefully fixed in https://github.com/PyvesB/AdvancedAchievements/commit/130efd4b5e95f5074d829a571b71e24a7b9a808c, let me know if you still encounter any problems.